client/src/hooks/auth.ts

```diff
@@ -23,7 +25,12 @@ const useAuth = () => {
       await csrf()
       const response = await axios.post('register', data)
       if (response.status === 204) {
-        toast.error('Account currently not yet verified', { position: 'top-right' })
+        toast.error('Account currently not yet verified please wait to the admin for approval.', {
+          position: 'top-right'
+        })
+        deleteCookie('XSRF-TOKEN')
+        redirect('/login')
```

Comment:
Remove `deleteCookie('XSRF-TOKEN')`, the backend will handle that one for you.
Use `window.location.href = 'login'` instead of `redirect`